src/nih_wayfinding/app/scripts/profiles/profile-service.js

```diff
@@ -65,27 +66,18 @@
          }
 
         /**
-         * @returns Profile for given username
+         * Return an empty user object
          */
-        function getProfile(name) {
-            return localStorageService.get(name);
+        function createBaseProfile() {
```

Comment:
This might be more clearly named as something like blankProfile, emptyProfile, or newProfile()?